### PR TITLE
CI for Node

### PR DESCRIPTION
**Pull request by HolkarA**
CI for Node

### CODE REVIEW

**Comment by HolkarA** [comment]
FAIL __test__/game.test.js

**Review by ManishOjwani — APPROVED** [review]

**Review by HolkarA — COMMENTED** [review]
All fine

**Review by pratap-mudaliar — APPROVED** [review]
Approved

**Review by pratap-mudaliar — APPROVED** [review]
Approved

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
```
[the remark this answers is not in the document]
Comment (pratap-mudaliar) [reply .github/workflows/node.js.yml:6]: approved

File: .github/workflows/node.js.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
```
[the remark this answers is not in the document]
Comment (pratap-mudaliar) [reply .github/workflows/node.js.yml:12]: Approved

File: .github/workflows/node.js.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x, 16.x]
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+
```
[the remark this answers is not in the document]
Comment (pratap-mudaliar) [reply .github/workflows/node.js.yml:21]: Approved

**Review by pratap-mudaliar — APPROVED** [review]

**Review by pratap-mudaliar — APPROVED** [review]

**Comment by ManishOjwani** [comment]
✕ Initializes with two players
✕ Starts the game with a random player
✕ Sets the current player to be whoever it is not

**Review by HolkarA — COMMENTED** [review]
fine

**Pull request merged**